pypeit/core/telluric.py

```diff
@@ -454,12 +598,13 @@ def tellfit(flux, thismask, arg_dict, init_from_last=None):
             This is actually two concatenated paramter vectors, one for
             the object and one for the telluric, i.e::
 
-                theta_obj = theta[:-7]
-                theta_tell = theta[-7:]
-
-            The telluric model theta_tell is currently hard wired to be six dimensional::
+                theta_obj = theta[:-(ntell+3)]
+                theta_tell = theta[-(ntell+3):]
 
-                pressure, temperature, humidity, airmass, resln, shift = theta_tell
+            The telluric model theta_tell includes a either user-specified
```

Comment:
Can you be more specific in the docs and indicate what parameters live at which indices in the theta vector.

Reply:
Done, I hope that was comprehensive enough...